demos/prefsFramework/js/prefsEditorDemo.js

```diff
@@ -102,6 +102,20 @@ var demo = demo || {};
         }
     });
 
+    // Fine-tune the syllabification schema
+    fluid.defaults("demo.prefsEditor.auxSchema.syllabification", {
+        gradeNames: ["fluid.prefs.auxSchema.syllabification"],
+        auxiliarySchema: {
+            syllabification: {
+                enactor: {
+                    terms: {
+                        patternPrefix: "../../src/lib/hypher/patterns"
```

Comment:
Remind me if we have any scheme for managing these kinds of relative file paths or whether the user is expected to override them for each enactor separately

Reply:
Sadly it is the integrators responsibility to override them.